File: anika.dovob/src/main/java/com/example/anika/homework/chapter5/entity/Member.java
package com.example.anika.homework.chapter5.entity;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity(name = "Member5")
@Table(name = "members")
@Getter
@Setter
@NoArgsConstructor
@ToString
public class Member {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "member_id")
    private Long id;

    private String name;

    @ManyToOne
    @JoinColumn(name = "team_id")
    private Team team;

    public Member(String name) {
        this.name = name;
    }
}

File: joon.bug/src/main/kotlin/com/example/joonbug/chapter6/homework1/Member.kt
package com.example.joonbug.chapter6.homework1

import jakarta.persistence.*

/**
 * 숙제 1: 일대다 단방향 - Member 엔티티
 *
 * 일대다 단방향이므로 Member는 Team을 참조하지 않음
 */
@Entity(name = "Hw1Member")
@Table(name = "hw1_member")
class Member(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,

    var name: String = ""
)